manuscript.qmd

```diff
@@ -228,19 +226,18 @@ side information. This side information can include
 sample attributes like treatment group, collection site, or time point. The
 ability to integrate heterogeneous data elements is essential in biological
 research, where the individual components of a system can be seldom studied in
-isolation. The challenge for a data scientist is that different methods often
-require variable and inconsistent input data formats, leading to technical
-complications and time lost on repeated data wrangling.
-Bioconductor addresses this through standardized data structures supported by
-an ecosystem of interoperable methods, allowing more time to be invested in core
-analytical tasks.
+isolation. However, input data formats vary across different methods, leading
+to technical complications and time lost on data wrangling.
+Bioconductor addresses this through standardized data structures supported
+by an ecosystem of interoperable methods, allowing more time to be invested
+into analytical tasks.
```

Comment:
"core analytical tasks" highlights that the data wrangling is just something that people must do but do not really advance the study